rocketpy/mathutils/function.py

```diff
@@ -1087,6 +1087,36 @@ def to_frequency_domain(self, lower, upper, sampling_frequency, remove_dc=True):
             extrapolation="zero",
         )
 
+    def low_pass_filter(self, alpha):
+        """
+        Implements a low pass filter with a moving average filter
+
+        Parameters:
+        - alpha (float): attenuation coefficient, 0 < alpha < 1
+
+        - for a given dataset, the larger alpha is, the more closely the filtered function returned will match the function
+        - the smaller alpha is, the smoother the filtered function returned will be (but with a phase shift)
+
+
+        Returns:
+        - filtered_function (Function): The function that filter the incoming source
+        """
+        filtered_signal = np.zeros_like(self.source)
+        filtered_signal[0] = self.source[0]
+
+        for i in range(1, len(self.source)):
+            filtered_signal[i] = (
+                alpha * self.source[i] + (1 - alpha) * filtered_signal[i - 1]
+            )  # for each point of our dataset, we apply a exponential smoothing
```

Comment:
I think this would be more convenient to read: 

```suggestion
             # for each point of our dataset, we apply a exponential smoothing
            filtered_signal[i] = (alpha * self.source[i] + (1 - alpha) * filtered_signal[i - 1]) 
```